core/src/test/java/it/pagopa/selfcare/mscore/core/DelegationServiceImplTest.java

```diff
@@ -254,4 +255,52 @@ void testCreateDelegationFromTaxCodeConflict() {
         assertThrows(ResourceConflictException.class, () -> delegationServiceImpl.createDelegationFromInstitutionsTaxCode(delegation));
     }
 
+    @Test
+    void testDeleteDelegationFromDelegationId() {
+        Delegation delegation = new Delegation();
+        delegation.setTo("id");
+        delegation.setStatus(DelegationState.DELETED);
+        when(delegationConnector.findByIdAndModifyStatus("id", DelegationState.DELETED)).thenReturn(delegation);
+        when(delegationConnector.checkIfDelegationsAreActive("id")).thenReturn(false);
+        Executable executable = () -> delegationServiceImpl.deleteDelegationFromDelegationId("id");
+        assertDoesNotThrow(executable);
+        verify(institutionService).updateInstitutionDelegation("id", false);
+    }
+
+    @Test
+    void testDeleteDelegationFromDelegationId2() {
```

Comment:
it might be better to name method based on what it is testing....for ex. testDeleteDelegationFromDelegationId_whenDelegationisActive()